.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
```

Comment:
Technical point -- is this gonna be global (both front and back ends)? Or will we have one per folder? In which case, amending this to `./client/node_modules/` would be great.